Documentation/gettingstarted/host-firewall.rst

```diff
@@ -6,13 +6,20 @@
 
 .. _host_firewall:
 
-*************
-Host Firewall
-*************
+******************************************
+Host Firewall (beta when using kube-proxy)
+******************************************
 
 This document serves as an introduction to Cilium's host firewall, to enforce
 security policies for Kubernetes nodes.
 
+.. note::
+
+    The host firewall is a beta feature when running without our kube-proxy
+    replacement. In particular, two bugs need to be addressed before we can
+    consider this feature stable: #12205 and #14859. Please provide feedback
```

Comment:
We now have a dedicated “role” for marking GitHub issue with Sphinx:

```suggestion
    consider this feature stable: :gh-issue:`12205` and :gh-issue:`14859`. Please provide feedback
```

Reply:
Oh, perfect! I was looking for a convention and surprised to find zero other references to GitHub issues :facepalm: